mitiq/pec/representations/biased_noise.py

```diff
@@ -84,20 +85,10 @@ def represent_operation_with_local_biased_noise(
     b = epsilon * (3 * eta + 1) / (3 * (eta + 1))
     c = epsilon / (3 * (eta + 1))
     alpha = (a**2 + a * b - 2 * c**2) / (
-        a**3
-        + a**2 * b
-        - a * b**2
-        - 4 * a * c**2
-        - b**3
-        + 4 * b * c**2
+        a**3 + a**2 * b - a * b**2 - 4 * a * c**2 - b**3 + 4 * b * c**2
     )
     beta = (-a * b - b**2 + 2 * c**2) / (
-        a**3
-        + a**2 * b
-        - a * b**2
-        - 4 * a * c**2
-        - b**3
-        + 4 * b * c**2
+        a**3 + a**2 * b - a * b**2 - 4 * a * c**2 - b**3 + 4 * b * c**2
```

Comment:
This line containing a lot of arithmetic was split into multiple lines, but is now a single line. I'm not saying this is incorrect, rather just wanted to confirm this is the behavior we want to see.